app/invocation/invocation_buttons.tsx

```diff
@@ -25,6 +32,12 @@ export default class InvocationButtons extends React.Component<InvocationButtons
     return repoUrl.startsWith("https://github.com/");
   }
 
+  private async onAskClicked() {
+    this.setState({ askLoading: true });
+    await this.props.model.fetchSuggestions();
+    this.setState({ askLoading: false });
+  }
```

Comment:
nit: can use `finally` so the `askLoading: false` line runs unconditionally

```suggestion
  private onAskClicked() {
    this.setState({ askLoading: true });
    return this.props.model.fetchSuggestions().finally(() => this.setState({ askLoading: false }));
  }
```

Reply:
Done